zubhub_frontend/zubhub/src/views/project_details/ProjectDetails.jsx

```diff
@@ -443,7 +450,52 @@ function ProjectDetails(props) {
           }
           aria-labelledby={t('projectDetails.ariaLabels.imageDialog')}
         >
-          <img className={classes.enlargedImageStyle} src={enlarged_image_url} alt={`${project.title}`} />
+          <IconButton
+            className={classes.cancelEnlargedImageBtn}
+            onClick={() =>
+              setState({
+                ...state,
+                open_enlarged_image_dialog: !open_enlarged_image_dialog,
+              })
+            }
+            aria-label={t('projectDetails.ariaLabels.closeImageDialog')}
+          >
+            <CloseIcon
+              className={classes.enlargedImageDialogCloseIcon}
+            />
+          </IconButton>
+          <Box className={classes.enlargedImageContainer}>
+            {project.images.length <= 1 ? (
+              <img
+                className={classes.enlargedImageStyle}
+                src={enlarged_image_url}
+                alt={`${project.title}`}
+              />
+            ) : (
+                <Slider
+                  initialSlide={enlarged_image_id}
+                  adaptiveHeight
+                  infinite
+                  speed={500}
+                  slidesToShow={1}
+                  slidesToScroll={1}
+                  className={classes.enlargedImageStyle}
+                  nextArrow={<EnlargedImgArrow />}
+                  prevArrow={<EnlargedImgArrow />}
```

Comment:
I don't know how the arrow looks, I am assuming they look like `<`, `>`. I think direction of arrow is missing

Reply:
React slick deals with the arrows themselves, the only styling I did is the background color, border radius of the circles etc. Using nextArrow, and prevArrow props automatically makes the arrows face the right directions.